addons/google_calendar/models/google_calendar.py

```diff
@@ -525,19 +516,20 @@ def remove_references(self, cr, uid, context=None):
             'google_calendar_cal_id': False,
         }
 
-        all_my_attendees = self.pool['calendar.attendee'].search(cr, uid, [('partner_id', '=', current_user.partner_id.id)], context=context)
-        self.pool['calendar.attendee'].write(cr, uid, all_my_attendees, {'oe_synchro_date': False, 'google_internal_event_id': False}, context=context)
+        all_my_attendees = self.env['calendar.attendee'].search([('partner_id', '=', current_user.partner_id.id)])
+        all_my_attendees.write({'oe_synchro_date': False, 'google_internal_event_id': False})
         current_user.write(reset_data)
         return True
 
-    def synchronize_events_cron(self, cr, uid, context=None):
-        ids = self.pool['res.users'].search(cr, uid, [('google_calendar_last_sync_date', '!=', False)], context=context)
+    @api.model
+    def synchronize_events_cron(self):
+        users = self.env['res.users'].search([('google_calendar_last_sync_date', '!=', False)])
         _logger.info("Calendar Synchro - Started by cron")
 
-        for user_to_sync in ids:
+        for user_to_sync in users.ids:
             _logger.info("Calendar Synchro - Starting synchronization for a new user [%s] " % user_to_sync)
             try:
-                resp = self.synchronize_events(cr, user_to_sync, False, lastSync=True, context=None)
+                resp = self.sudo(user_to_sync).synchronize_events(False, lastSync=True)
```

Comment:
Yes, As I said below, there is no need of ids at all and hence there is no need of False